core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -147,35 +218,59 @@ var LearnerDashboardPage = function() {
     expect(await feedbackMessage.first().getText()).toMatch(message);
   };
 
-  this.checkIncompleteExplorationSection = async function(explorationTitle) {
-    await this.navigateToInCompleteSection();
-    await this.navigateToIncompleteExplorationsSection();
-    await this.expectTitleOfExplorationSummaryTileToMatch(
-      explorationTitle);
-  };
+  this.navigateToCommunityLessonsAndCheckIncompleteExplorations = (
+    async function(explorationTitle) {
+      await this.navigateToCommunityLessonsSection();
+      // eslint-disable-next-line oppia/check-element-selector-at-top
+      var explorationTitleInIncompleteSection = element(
+        by.css(
+          '.protractor-test-incomplete-community-lessons-section')).element(
+        by.cssContainingText(
+          '.protractor-test-exp-summary-tile-title', explorationTitle));
```

Comment:
The first element selector really should go at the top. You can do something like this:

```js
// At the top of the page
var incompleteCommunityLessonsSection = element(by.css('.protractor-test-incomplete-community-lessons-section'));
...
// In `navigateToCommunityLessonsAndCheckIncompleteExplorations`
var explorationTitleInIncompleteSection = incompleteCommunityLessonsSection.element(by.cssContainingText(...));
```

Sorry I didn't explain this in my earlier comment. I forgot that we had added this lint check.

ditto elsewhere

Reply:
Done.